types/react-hyperscript-helpers/index.d.ts

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable no-unused-vars */
+
+import type * as React from 'react'
+
+
+declare module 'react-hyperscript-helpers' {
+  type Component<P> = React.FunctionComponent<P> | React.ComponentClass<P>
+
+  type Children<Props> = Props extends { children?: React.ReactNode } ? React.ReactNode[] : (Props extends { children?: unknown } ? [Props['children']] : never)
+
+  interface hHelper {
```

Comment:
all types and interfaces should start with a CapitalLetter

Reply:
Changed in 2c5e30dc49f5585ecaadbce167ae76f2000a185a.